test/core/2_pendleData.spec.ts

```diff
@@ -43,11 +43,11 @@ describe('PendleData', async () => {
 
   it('Should be able to deploy with correct initial data', async () => {
     const treasuryInData = await data.treasury();
-    expect(treasuryInData).to.be.eq(treasury.address);
+    expect(treasuryInData.toLowerCase()).to.be.eq(treasury.address.toLowerCase());
```

Comment:
Do write a function (maybe into helpers) to compare two addresses without case sensitive and use it instead of this.